target/linux/mediatek/dts/mt7981b-cudy-tr3000-v1.dts

```diff
@@ -0,0 +1,231 @@
+// SPDX-License-Identifier: (GPL-2.0 OR MIT)
+
+/dts-v1/;
+
+#include <dt-bindings/leds/common.h>
+
+#include "mt7981.dtsi"
+
+/ {
+        model = "Cudy TR3000 v1";
```

Comment:
Please use tabs instead of spaces for indentation in dts.

Reply:
Done. My original file had tabs, but they were converted to spaces by copying and pasting from the terminal.